### PR TITLE
Midround Event changes

### PR DESCRIPTION
**Pull request by Cw3040**
Changelogs in the Commits my friends

### CODE REVIEW

**Comment by Cw3040** [comment]
Wait this fucked up

**Comment by CameronWoof** [comment]
Sure.

**Review by deathride58 — DISMISSED** [review]
Modularize these changes by overriding them in modular_citadel or add `//CITADEL CHANGE - adjusts x to y` comments to every single line you've changed

**Review by deathride58 — CHANGES_REQUESTED** [review]
hec wrong button

File: modular_citadel/code/modules/events/devil.dm [hunk 1]
```diff
@@ -0,0 +1,61 @@
+/datum/round_event_control/devil
```
Comment (silicons) [inline modular_citadel/code/modules/events/devil.dm]: No devil.
Just no.

**Review by silicons — CHANGES_REQUESTED** [review]
Therei s no possible way you can convince me spawning a devil without a crazy admin doing it is a good idea.

**Review by deathride58 — CHANGES_REQUESTED** [review]
Add the new files to the .dme, and make sure the files only override the variables you've changed

**Comment by Cw3040** [comment]
alright time to quit doing this, my position is clearly in the map world

**Pull request closed without merging**